cypress/integration/common/menu.feature

```diff
@@ -119,26 +119,30 @@ Feature: Menu component
     When I click closeIcon
     Then Menu is in fullscreen mode is not visible
 
-  @positive
+  @ignore
```

Comment:
Please can you add the JIRA ticket reference for the new task to these comments.